Update chapter 4 notes

mbrown118 opened the pull request:
add notes on chapter 4

mbrown118 committed:
start ch 4 notes

mbrown118 committed:
added in more LaTeX and fixed long fig captions

mbrown118 committed:
added 3 figures

mbrown118 committed:
notes on empirical comparison, poisson regression and GLM

mbrown118 committed:
add logistic regression formulas

mbrown118 committed:
add prediction formula

mbrown118 committed:
Merge branch 'main' of https://github.com/mbrown118/bookclub-islp

mbrown118 committed:
added some extra details

mbrown118 committed:
create 04_presentation.qmd

jonthegeek committed:
Merge branch 'main' into main

jonthegeek commented on pyproject.toml:
```diff
@@ -22,3 +22,6 @@ dependencies = [
 ]
+
+[tool.uv.workspace]
+members = ["Users/brown/Docments/dev/environments/islp"]
```
This looks like a path on your machine. What is this supposed to do?

jonthegeek replied:
@mbrown118 Let me know what this is for, so we can figure out what's supposed to be happening and get this merged into the slides!

mbrown118 replied:
I removed it! It was just for my own personal package/environment management.

jonthegeek reviewed (changes requested):
It seems like a lot of files, but if this is what it takes, that's fine! My only concern is the path you added to pyproject.toml. I don't really know anything about the Python side of things, but that looks like a path that's local to you, so I don't think it should be in the general slides. If it does something useful for you, it's fine to stay there, but I suspect it isn't doing what's expected right now.

mbrown118 committed:
embed resources on 04_presentation.qmd

mbrown118 committed:
Merge branch 'main' of https://github.com/mbrown118/bookclub-islp

mbrown118 committed:
remove directory from pyproject.toml

mbrown118 commented:
I removed the part of pyproject.toml with my directory and I removed many of the extra files by adding `embed-resources: true` to the qmd file. Thank you!

jonthegeek reviewed (approved):
Fantastic, thanks!
